### PR TITLE
feature/webstore-029

### PR DESCRIPTION
**Pull request by alexMIRON1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,4 +62,19 @@ configure to devtools to do it. For it
...
 Then, your project will be automatically restarted in runtime after changes.
 
-> I would not suggest to use second option, because of unstable.
\ No newline at end of file
+> I would not suggest to use second option, because of unstable.
+## Set up localstack
+
+Create local localstack
+```
+docker run --name local-localstack-container --restart=on-failure -p 4566:4566 localstack/localstack:latest
```
Comment (sokimaaa) [inline README.md:70]: 1) with such command you are using all of services provided by localstack, but we need only SQS, SNS
to take only services we need we could use env vars provided by localstack, e.g. SERVICES=sqs

2) actually, I would suggest to take a look at another env vars provided by localstack, maybe you find something useful for us.

3) also, to use localstack in the our app we need 'localstack profile', to trigger configuring it:
aws configure --profile localstack
update configure with its

4) lets use for localstack's services ports 8010 and 8011 for SQS and SNS accordingly

5) Did you test all work correctly? (I mean, create, delete queues, push and pull messages from them)

6) Please, add a bunch of useful CLI - commands to interact with queues via CLI (probably you'll have to install AWS CLI before)

7) Add explanation what is topic in SQS, probably newcomers won't know what it is

File: README.md [hunk 2]
```diff
@@ -62,4 +62,19 @@ configure to devtools to do it. For it
 
 Then, your project will be automatically restarted in runtime after changes.
 
-> I would not suggest to use second option, because of unstable.
\ No newline at end of file
+> I would not suggest to use second option, because of unstable.
+## Set up localstack
```
Comment (sokimaaa) [inline README.md:66]: Actually we are setting up the SQS, SNS services via Localstack

File: README.md [hunk 3]
```diff
@@ -62,4 +62,19 @@ configure to devtools to do it. For it
...
+Create local localstack
+```
+docker run --name local-localstack-container --restart=on-failure -p 4566:4566 localstack/localstack:latest 
+```
+Set up SNS service and create sns topic.
+Also, set up SQS service and create sqs queue using CLI.
+```
+docker exec -it local-localstack-container /bin/bash
+
+awslocal sqs create-queue --queue-name webstore-queue
```
Comment (sokimaaa) [inline README.md:77]: could you please make queue more configured? (e.g. we want to use FIFO queue, endpoint url, timeouts, output format and et cetera)

File: README.md (hunk 1 above)
Comment (sokimaaa) [reply README.md:70]: 8. Lets use exact version of localstack to avoid inconsistency between developers.

File: README.md (hunk 2 above)
Comment (alexMIRON1) [reply README.md:66]: Yes, but we exec localstack container through CLI.

**Review by sokimaaa — APPROVED** [review]

**Pull request merged**